### ISSUE TITLE
<LS> and <PS> are valid characters in string literals

### ISSUE THREAD

**Issue by ascidian** [tc39/test262-parser-tests#26]
Per specification https://tc39.es/ecma262/#prod-StringLiteral, &lt;LS&gt; and &lt;PS&gt; are valid characters in string literals

The following two tests must pass:
   fail/647e21f8f157c338.js
   fail/8af69d8f15295ed2.js

**Comment by kumavis** [comment]
`eval("'\u2028'")` executes without error in a quick test on v8 and jsc

**Comment by kumavis** [comment]
I agree with @ascidian's interpretation of the spec that these characters should be allowed

the spec for [`DoubleStringCharacter`](https://tc39.es/ecma262/#prod-DoubleStringCharacter) starts with
> SourceCharacter but not one of " or \ or LineTerminator

and while [`LineTerminator`](https://tc39.es/ecma262/#prod-LineTerminator) includes: 
```
U+000A     LINE FEED (LF)     <LF>
U+000D     CARRIAGE RETURN (CR)     <CR>
U+2028     LINE SEPARATOR     <LS>
U+2029     PARAGRAPH SEPARATOR     <PS> 
```
the spec continues to explicitly allow
```
<LS>
<PS>
```

**Comment by kumavis** [comment]
here is an example of `<LS>` in the wild (albeit in json) https://npmfs.com/package/unicode-confusables/0.1.1/data/confusables.json#L160

**Comment by michaelficarra** [comment]
Fixed by #30.
